contracts/interfaces/ISuper1155.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.7;
+
+import "../libraries/DFStorage.sol";
+
+
+interface ISuper1155 {
+  
+    function getThisMetadataUri() external view returns (string memory);
```

Comment:
Why is this function defined? The `metadataUri` variable is already public so we should just use its implicit getter.
```js
function metadataUri() external view returns (string memory);
```